test/common/testlib.py

```diff
@@ -565,6 +565,11 @@ def _key_press_firefox(self, keys: str, modifiers: int = 0, use_ord: bool = Fals
             args["type"] = "keyUp"
             self.cdp.invoke("Input.dispatchKeyEvent", **args)
 
+    # FIXME: This should be key_press(), and key_press() should be something else
+    def key(self, name: str) -> None:
+        self.cdp.invoke("Input.dispatchKeyEvent", type="keyDown", key=name)
+        self.cdp.invoke("Input.dispatchKeyEvent", type="keyUp", key=name)
```

Comment:
`key` sounds like a good compromise, breaking `key_press` is imo a no-go at this moment as everyone uses it.

Reply:
Yes, I thought about `.key()` and `.input_text()`, moving everything over to these two, and eventually dropping `.key_press()`